percona-server-8.3.yaml

```diff
@@ -0,0 +1,125 @@
+package:
+  name: percona-server-8.3
+  version: 8.3.0
+  epoch: 0
+  description: "Percona Server for MySQL is a free, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It provides superior performance, scalability, and instrumentation."
+  copyright:
+    - license: GPL-3.0-or-later
+
+environment:
+  contents:
+    packages:
+      - bison
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - curl-dev
+      - libaio-dev
+      - libedit
+      - libevent-dev
+      - libtirpc-dev
+      - linux-pam-dev
+      - ncurses-dev
+      - nfs-utils
+      - openldap-dev
+      - openssl-dev
+      - pcre2-dev
+      - perl
+      - readline-dev
+      - samurai
+      - sed
+      - wolfi-baselayout
+      - xz-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/percona/percona-server
+      tag: Percona-Server-${{package.version}}-1
+      expected-commit: fcee26ff5d51b9e72fa8b3b626a3424e2cbe2794
+
+  - name: "Update Submodules"
+    runs: |
+      git submodule update --init --recursive
+
+  - name: "Remove Coredumper Support"
+    runs: |
+      sed -i 's/INCLUDE(coredumper)//g' CMakeLists.txt
+      sed -i 's/MYSQL_CHECK_COREDUMPER()//g' CMakeLists.txt
+      sed -i 's/OPTION (WITH_COREDUMPER "Enable support for coredumper library" ON)//g' CMakeLists.txt
+
+  - name: "Cmake"
+    runs: |
+      mkdir /var/tmp
+      cmake . -G Ninja \
+        -DFORCE_INSOURCE_BUILD=1 \
+        -DBUILD_CONFIG=mysql_release \
+        -DWITH_EDITLINE=system \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DSYSCONFDIR=/etc \
+        -DMYSQL_DATADIR=/var/lib/mysql \
+        -DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock \
+        -DDEFAULT_CHARSET=utf8mb4 \
+        -DDEFAULT_COLLATION=utf8mb4_general_ci \
+        -DENABLED_LOCAL_INFILE=ON \
+        -DINSTALL_INFODIR=share/info \
+        -DINSTALL_MANDIR=share/man \
+        -DINSTALL_PLUGINDIR=lib/${{package.name}}/plugin \
+        -DINSTALL_INCLUDEDIR=include/mysql \
+        -DINSTALL_DOCREADMEDIR=share/doc/${{package.name}} \
+        -DINSTALL_SUPPORTFILESDIR=share/${{package.name}} \
+        -DINSTALL_DOCDIR=share/doc/${{package.name}} \
+        -DWITH_ROCKSDB=NO \
+        -DTMPDIR=/var/tmp \
+        -DWITH_ASAN=OFF \
+        -DWITH_EXTRA_CHARSETS=complex \
+        -DWITH_SYSTEMD=no \
+        -DWITH_SSL=system \
+        -DWITH_VALGRIND=OFF \
+        -DWITH_ZLIB=system \
+
+  - name: "Install"
+    runs: |
+      DESTDIR="${{targets.destdir}}" ninja install
```

Comment:
The various COPYING.* licenses state they need to be included in any source or binary distribution - are those in the resulting APK file?

Reply:
Where would be a good place to put them? can I just put them in a licenses folder in the build?

Reply:
Added licensing